### PR TITLE
VSFP-18 research & analyze enso

### PR DESCRIPTION
**Pull request by lstreckeisen**

### CODE REVIEW

File: SA/bibliography.bib [hunk 1]
```diff
@@ -1,3 +1,9 @@
+@online{enso-language,
+	author = {{Enso International Inc}},
```
Comment (fl3pp) [inline SA/bibliography.bib:2]: Are the nested `{` by intentions?

File: SA/bibliography.bib [hunk 2]
```diff
@@ -9,11 +15,11 @@ @manual{snap-manual
 	year = {2020}}
 
 @techreport{Elliot_2007,
-  author      = {Elliot, Conal M.},
-  title       = {Tangible Functional Programming},
-  year        = {2007},
-  month       = {Oct},
-  url         = {http://conal.net/papers/Eros/eros.pdf},
+    author      = {Elliot, Conal M.},
```
Comment (fl3pp) [inline SA/bibliography.bib:18]: inconsistent indentation on this line

File: SA/design_concept/content/tool_research/enso.typ [hunk 3]
```diff
@@ -0,0 +1,16 @@
+= Enso
+Enso is a functional programming language created by Enso International Inc., designed for data science.
+Programs can be created by both text and a visual editor.
+The visual editor allows a user to define components that can be connected with each other, symbolizing the data-flow from one component to another.
```
Comment (fl3pp) [inline SA/design_concept/content/tool_research/enso.typ:4]: Maybe add an empty line in here somewhere to create 2 paragraphs and break the block.

File: SA/design_concept/content/tool_research/enso.typ [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+The editor also offers previews for a component's data, e.g. allowing a user to see a modified picture like in @enso_screenshot.
```
Comment (fl3pp) [inline SA/design_concept/content/tool_research/enso.typ:5]:
```suggestion
The editor also offers previews for a component's data, which e.g. allows a user to see a modified picture like in @enso_screenshot.
```

File: SA/design_concept/content/tool_research/enso.typ [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+@enso-language
```
Comment (fl3pp) [inline SA/design_concept/content/tool_research/enso.typ:6]:
```suggestion
```

File: SA/design_concept/content/tool_research/enso.typ [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+#figure(
+    image("../../static/enso_screenshot.png", width: 80%),
+    caption: [Example programm in Enso @enso-language]
+)<enso_screenshot>
+
+Enso is visually impressive and largely intuitive reagarding data-flow.
+E.g. downloading data from a public API and aggregating it is super easy.
+However, e.g. dividing a number with another number is somehow quite complicated.
```
Comment (fl3pp) [inline SA/design_concept/content/tool_research/enso.typ:15]:
```suggestion
However, some operations, such as dividing a number with another number, are somehow quite complicated.
```

File: SA/bibliography.bib (hunk 1 above)
Comment (lstreckeisen) [reply SA/bibliography.bib:2]: The nested `{}` prevents the author name form being shortened in the bibliography

Comment (fl3pp) [reply SA/bibliography.bib:2]: hu, ok, I didn't know that 👍

File: SA/design_concept/content/tool_research/enso.typ (hunk 5 above)
Comment (fl3pp) [reply SA/design_concept/content/tool_research/enso.typ:6]: Now I see, that reference is meant for the complete paragraph.

Maybe put it a bit higher and reference it in the text?

Comment (lstreckeisen) [reply SA/design_concept/content/tool_research/enso.typ:6]: I'd like to leave this in, to make it clear that that paragraphy is based on information from enso, and the text paragraph on my impressions.

**Comment by fl3pp** [comment]
approved 😅✅

**Review by fl3pp — APPROVED** [review]

**Pull request merged**